packages/manager/modules/dedicated-universe-components/src/components/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+
+import user from './user';
+import noDefaultMeansOfPayment from './no-default-means-of-payment';
+import stepper from './stepper';
+import OvhHttp from './ovh-angular-http/ovh-angular-http.module';
```

Comment:
Why not import it in the submodule that needs the dependency ? It doesn't seem needed at the root level